src/script/server/api/services/state/state.js

```diff
@@ -0,0 +1,21 @@
+const reducers = require('../../../../redux/reducers')
+const { connect } = require('../../../../redux/connect')
+const db = require('../db')
+
+const getFileName = connect(() => (dispatch, getState) => {
+  const component = reducers.component.get()(getState())
+  return `${component.name}.json`
+})
+
+const create = async (state) => {
+  await db().write(getFileName(), state)
+}
+
+const read = async () => {
+  return await db().read(getFileName()) || {}
```

Comment:
you can write it like that : 
const read = async () => db().read(getFileName())